zebra/zebra_rib.c

```diff
@@ -980,7 +959,7 @@ static int nexthop_active_update(struct route_node *rn, struct route_entry *re,
 		 * a multipath perpsective should not be a data plane
```

Comment:
There's another odd comment just above this point - claiming that we don't care about source address, but it does seem to be part of the big test at the end of each iteration of this loop?

Reply:
I'm not reading it that way, I read it as RE's as received from an upper level protocol never get the source specified, so we don't track the src as something that modifies *this* nexthop, since it may but not here.  I'm not going to touch this code at the moment since I didn't modify anything here :)